views/js/qtiCreator/helper/devTools.js

```diff
@@ -19,7 +19,9 @@ define([
     'lodash',
     'jquery',
     'taoQtiItem/qtiCreator/helper/xmlRenderer',
-    'taoQtiItem/qtiCreator/model/helper/event'
+    'taoQtiItem/qtiCreator/model/helper/event',
+    'taoQtiItem/qtiXmlRenderer/test/renderer/vkBeautify',
+    'taoQtiItem/qtiXmlRenderer/test/renderer/prism/prism'
```

Comment:
It was not directly related to issue, but that missing dependencies prevented to run xml debugging in widgets here https://github.com/oat-sa/extension-tao-itemqti/blob/50a9a3cb1f92c4f63a082f4bce03906ae2c1c62a/views/js/qtiCreator/widgets/item/Widget.js#L414-L414